code/game/objects/items/plushes.dm

```diff
@@ -66,8 +107,143 @@
 		var/turf/T = get_turf(user)
 		log_game("[key_name(user)] added a grenade ([I.name]) to [src] at [COORD(T)].")
 		return
+	if(istype(I, /obj/item/toy/plush))
+		love(I, user)
+		return
 	return ..()
 
+/obj/item/toy/plush/proc/love(obj/item/toy/plush/Kisser, mob/living/user)	//~<3
+	var/chance = 100	//to steal a kiss, surely there's a 100% chance no-one would reject a plush such as I?
+	var/concern = 20	//perhaps something might cloud true love with doubt
+	var/loyalty = 30	//why should another get between us?
+	var/duty = 50		//conquering another's is what I live for
+
+	//we are not catholic
+	if (young == TRUE || Kisser.young == TRUE)
+		user.show_message("<span class='notice'>[src] plays tag with [Kisser].</span>", 1,
+			"<span class='notice'>They're happy.</span>", 0)
```

Comment:
*single tear*

Reply:
Really, please remove this. This will attract the worst possible kind of players to our servers.

Reply:
I can't tell if this is sarcasm or not, help me out